system/content-directory/nft.md

```diff
@@ -98,6 +98,26 @@ An _NFT_ represents ownership title over video, and it is defined by the followi
 * **Status: The**  representing the state of the NFT currently.
 * **Royalty:** If set, it specifies the fraction of the paid value of later transactions which must accrue to the issuer.
 
+### Minting Limits
+Minting limits on NFTs are applied to avoid massive proliferation of low quality NFT projects.
+
+There are several types of limits, depending on the time window considered and context:
+
+**Global Limits**
+- Global weekly limit: max number of NFT that can be minted each week, stored on chain 
+- Global daily limit: max number of NFT that can be minted each day, stored on chain
+
+The above values are updated via a `UpdateGlobalNftLimit` proposal.
```

Comment:
Can you link to this proposal in the proposal article?

Reply:
👍